pages/common/mscore.md

```diff
@@ -1,6 +1,7 @@
 # mscore
 
 > This command is an alias of `musescore`.
+> More information: <https://musescore.org/en/handbook/3/command-line-options>.
```

Comment:
We can remove language and version, maybe it's always redirected to the latest version.
```suggestion
> More information: <https://musescore.org/handbook/command-line-options>.
```